src/vale.adoc

```diff
@@ -24,3 +24,98 @@ Some of the errors that the AsciiDoc rules highlight include:
 
 The RISC-V style for Vale enforces the rules that are set by the RISC-V style guidelines. 
 
+=== Abbreviations
+
+Do not use punctuation in abbreviations.
+
+=== Anthropomorphism
+
+Anthropomorphism is granting abilities to inanimate objects, such as "This code allows you to enable access." Instead, make the user the focus of the sentence: "You can use this code to enable access." 
+
+=== Camel case
+
+If you are using Camel case (`camelCase`), consider using backticks.
+
+=== Case sensitive terms
+
+Terms such as RISC-V are case sensitive. If your usage is in code, then be sure to place back ticks around the code phrase.
+
+=== Conjunctions
+
+Do not overuse conjunctions when you begin a sentence, including the following conjunctions:
+
+* And
+* But
+* Or
+* So
+
+=== Contractions
+
+Avoid using contractions in Extension documentation. Contractions can cause ambiguity with translations.
```

Comment:
"Avoid using" or "Don't use" (haha!) "Do not use"?

Same for other similar cases below. Are we stating rules, or guidelines? Does Vale know the difference?

Is there a reason "Extension" is capitalized?